From 5ebdf436dfbed36c0ace433c949ef841d93dfea8 Mon Sep 17 00:00:00 2001
From: Kris Shannon <kris@shannon.id.au>
Date: Sun, 17 Jul 2016 16:37:20 +1000
Subject: [PATCH] joyent/node-triton-tags#1 Port range error message doesn't
 match the code Reviewed by: Alex Wilson <alex.wilson@joyent.com> Reviewed by:
 Cody Mello <cody.mello@joyent.com> Approved by: Cody Mello
 <cody.mello@joyent.com>

---
 lib/index.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/index.js b/lib/index.js
index a6f543f..d09c384 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -83,7 +83,7 @@ var validatorFromKey = {
                 if (isNaN(svc.port) || svc.port < 1 || svc.port > 65535) {
                     return format(
                         'invalid "triton.cns.services" tag: service port ' +
-                        'number for %s must be within the range 0 - 65535',
+                        'number for %s must be within the range 1 - 65535',
                         svc.name);
                 }
             }
-- 
2.21.0

